server/base/src/main/java/org/apache/accumulo/server/util/RestoreZookeeper.java

```diff
@@ -118,6 +118,9 @@ public static void main(String[] args) throws Exception {
     }
 
     SAXParserFactory factory = SAXParserFactory.newInstance();
+    factory.setFeature("http://xml.org/sax/features/external-general-entities", false);
+    factory.setFeature("http://xml.org/sax/features/external-parameter-entities", false);
+    factory.setFeature("http://apache.org/xml/features/nonvalidating/load-external-dtd", false);
```

Comment:
> So, I would just do the following instead of anything more complicated or speculative with respect to a particular parser:

Fantastic. This is exactly the kind of answer I was expecting to hear from you :)

> But, this tool is typically run in the same security context as the user... and not by a remote service, so there really isn't a vulnerability out-of-the-box anyway.

Agreed. That's why I didn't go the route of private file+fix+announce. It's a nice thing we can do to make sure users don't do anything "dumb", but it's not a vulnerability in the traditional sense of XEE.